terraform/040-id-broker/vars.tf

```diff
@@ -172,10 +172,57 @@ variable "desired_count" {
   default = "1"
 }
 
+variable "email_signature" {
+  type    = "string"
+  default = ""
+}
+
+variable "password_forgot_url" {
+  type = "string"
+}
+
 variable "password_profile_url" {
   type = "string"
 }
 
+variable "support_email" {
+  type = "string"
+}
+
+variable "support_name" {
```

Comment:
You could add a `default = ""` (or `default = "support"`) here if you want, since this will internally default to `'support'` if empty.

Reply:
sure, I'll go with default of "support"